kafka_offset_migrate/main.py

```diff
@@ -0,0 +1,104 @@
+import argparse
+import logging
+import sys
+
+from kafka_offset_migrate.kafka_client import KafkaClient
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def parse_command_line():
+    parser = argparse.ArgumentParser(description='Simple tool to migrate '
+                                                 'offsets between kafkas.')
+
+    parser.add_argument('-s', '--src-kafka', default='localhost:9092',
+                        help='The kafka adresse list from where get ofssets.')
```

Comment:
@Evnsan, some entries here have the same typos that was on readme, and there is more of this in readme. Maybe it's better a search/replace on project.
```suggestion
                        help='The kafka adress list from where get offsets.')
```

Reply:
Done